api.js

```diff
@@ -27,3 +27,38 @@ export const fetchCommentsByArticleId = (article_id) => {
       throw err;
     });
 };
+export const postComment = (article_id, comment) => {
+  return myApi
+    .post(`/articles/${article_id}/comments`, {
+      body: comment.body,
+      username: "grumpy19",
+    })
+    .then((res) => {
+      return res.data;
+    })
+    .catch((err) => {
+      throw err;
+    });
+};
+export const fetchAlUsers = () => {
+  return myApi
+    .get("/users")
+    .then((res) => {
+      return res.data;
+    })
+    .catch((err) => {
+      throw err;
```

Comment:
you shouldn't be throwing errors here, have a read of [this article](https://dev.to/swyx/errors-are-not-exceptional-1g0b) if you're interested in when you should use throw